sdk/ios/README.md

```diff
@@ -42,15 +42,14 @@ Auth.auth().createUser(withIProovUserID: "hello@world.com") { result, error in
 }
 ```
 
-To sign in an existing user, simply use `Auth.auth().signIn(withIProovUserID:)` and pass their `userID`.
+To sign in an existing user, simply use `Auth.iProov().signIn(withUserID:)` and pass their `userID`.
 
 ### Advanced example
 
 Once you've got up and running with the basic example, you can now pass additional parameters to `createIProovUser()` and `signInIProovUser()`:
```

Comment:
`createIProovUser()` and `signInIProovUser` should be `createUser()` and `signIn()`.

**OR** referred to as `Auth.iProov().createUser()` and `Auth.iProov().signIn()`